src/rgw/rgw_aio.cc

```diff
@@ -58,7 +58,7 @@ Aio::OpFunc aio_abstract(librados::IoCtx ctx, Op&& op, jspan_context* trace_ctx
         (void)trace_ctx; // suppress unused trace_ctx warning. until we will support the read op trace
```

Comment:
This reverts 9f160e4f1aa385f39f1cd55eb42f278c94dfd85c has bee made on top of (depends on) 5d4c5e65be94a346e62b9f8ac63241779c4c7a6a.

**NOTE for the future** (_please don't solve this comment_): we can't forget abort bringing it back after fixing the encoding issue.